packages/lesswrong/themes/defaultPalette.ts

```diff
@@ -196,7 +196,7 @@ export const defaultComponentPalette = (shades: ThemeShadePalette): ThemeCompone
     primaryAlert: "#69886e",
     error: "#9b5e5e",
     error2: "#E04E4B",
-    warning: "#832013",
+    warning: "#621919",
```

Comment:
Oh, sorry I plan to revert that (I ended up fully decoupling the LW rendering from the WarningBanner).

(the reason LW is changing the color is so that rate limits don't feel as punishing/hostile)